docs/end-user-api.md

```diff
@@ -725,6 +728,22 @@ We have created a macro named `current`, which will always be translated to the
 
 There is also a similar macro named `next` which resolves to the beginning of the next interval. For example, if your time grain is `day`, then, `next` will resolve to tomorrow's date.
 
+#### Time Zone
```

Comment:
This should probably have `####` on the other side of `Time Zone` as well, for consistency if nothing else.

/NotABlocker